core-job/src/main/java/org/apache/kylin/job/execution/ExecuteResult.java

```diff
@@ -30,6 +30,14 @@
 
     private final State state;
     private final String output;
+    private final Throwable throwable;
+
+    /**
+     * Default constructor to indicate a success ExecuteResult.
+     */
+    public ExecuteResult() {
```

Comment:
Better to use static to create instance with state, and keep constructor generic, for example:
ExecuteResult.createSucceeded()
ExecuteResult.createError()
...

Reply:
Make Sense, will refine this part.